tests/test_restaurants_api.py

```diff
@@ -1,28 +1,47 @@
 # coding: utf-8
 
+from typing import Dict, List, Union, Tuple
+
 from fastapi.testclient import TestClient
 
 
 from openapi_server.models.list_restaurants import ListRestaurants  # noqa: F401
 from openapi_server.models.list_reviews import ListReviews  # noqa: F401
+from openapi_server.main import app
 
 
 def test_get_restaurants(client: TestClient):
```

Comment:
will it make a real request to googleapi? we would need to mock this google api call.

Reply:
It does make a real request. If I mock the api call, would I need to comment out parts of `get_restaurants` in `restaurants_api.py`?

Reply:
You should mock AsyncClient in get_restaurants method to return the mocked response. So get_restaurants method works without concerning about the dependency and focus on the method logic.

Reply:
So only upon deployment we replace the mock api call with the actual google api call?

Reply:
I just discovered that the mock AsyncClient only needs to be defined in the **test file** where the `patch` context manager from the built-in unittest library can handle replacing the responses from AsyncClient at the endpoint being tested with the `patch` decorator. To answer my own question, no code needs to be commented/replaced at any point.